src/get-data.ts

```diff
@@ -40,10 +46,13 @@ const getData = async (): Promise<PRData> => {
   if (info.repository && info.sender) {
     prData.actor = info.sender.login;
   } else {
-    throw new Error('Issue with context');
+    if (!info.repository && !info.sender) throw new Error('repository and sender are undefined');
```

Comment:
could info itself ever be undefined? or the context variable?

Reply:
if the context.payload is null yes. this function is called in index which is wrapped in a try catch which if failed will print that the action failed to run

Reply:
This should be an if/else if/else if instead of 3 ifs and you should always include {}